plugins/poll/plugin.rb

```diff
@@ -85,8 +85,6 @@ def vote(post_id, poll_name, options, user)
           available_options = poll.poll_options.map { |o| o.digest }.to_set
           options.select! { |o| available_options.include?(o) }
 
-          raise StandardError.new I18n.t("poll.requires_at_least_1_valid_option") if options.empty?
```

Comment:
I think we still need this since we should be using a different route for the vote removal